knowledge-base/how-to-capture-traffic-from-another-pc.md

```diff
@@ -26,35 +26,42 @@ How to capture HTTP/HTTPS traffic from a client on the same network as the Fiddl
 
 The Fiddler Everywhere proxy can capture HTTP/HTTPS traffic from virtually any device (like a computer, smartphone, smart tv, etc.) on the same network as the Fiddler Everywhere host. Before configuring the proxy on the client machine and the host, you must consider the following prerequisites:
 
-- The Fiddler Everywhere host and the client using the Fiddler Everywhere proxy must be on the same network.
-- The Fiddler Everywhere host and the client using the Fiddler Everywhere proxy must be discoverable on the network. There must be no administrative limitations or security policies that prevent device discovery on the local network.
+- The Fiddler Everywhere host and the target using the Fiddler Everywhere proxy must be on the same network.
 
->tip If you can ping the Fiddler host from the client device and the other way around, that would indicate that both devices are "seeing" each other.
+- The Fiddler Everywhere host and the target using the Fiddler Everywhere proxy must be discoverable on the network. There must be no administrative limitations or security policies that prevent device discovery on the local network.
+
+    >tip If you can ping the Fiddler host from the target device and the other way around, that would indicate that both devices are "seeing" each other.
+
+- The Fiddler Everywhere host and the target using the Fiddler Everywhere proxy should have the Fiddler proxy port opened. By default, the port is **8866**.
+
+
+The requirement for using the same network is actually not a must. However, most remote networks, like those managed by ISPs, use firewalls and explicitly close most ports. That makes configuring and using Fiddler over the internet virtually impossible (as there are not many ISPs that will allow you to open a random port). If you are the administrator of two separate networks, you must ensure that the Fiddler proxy port is open and that both host and target machines are seeing each other (successful ping).
 
 ## Configuring Fiddler Everywhere (Host) 
```

Comment:
We must note that Fiddler's root certificate must be trusted and HTTPS capturing to be enabled on the machine where Fiddler Everywhere is running. We are going to remove this limitation in the future, but we have it for the moment.